src/FMI2/cconst.jl

```diff
@@ -20,35 +20,44 @@ const fmi2Real = Creal      # defined in FMICore.jl, dependent on the Julia arch
 const fmi2Integer = Cint
 const fmi2Byte = Char
 const fmi2ValueReference = Cuint
+
+"""
+fmi2FMUstate is a pointer to a data structure in the FMU that saves the internal FMU state of the actual or
+a previous time instant. This allows to restart a simulation from a previous FMU state.
+
+Source: FMI2.0.3 Spec [p.17]; See also section 2.1.8
+"""
 const fmi2FMUstate = Ptr{Cvoid}
 const fmi2Component = Ptr{Cvoid}
 const fmi2ComponentEnvironment = Ptr{Cvoid}
 const fmi2Enum = Array{Array{String}} # TODO: remove, this is not part of the spec!
 export fmi2Char, fmi2String, fmi2Boolean, fmi2Real, fmi2Integer, fmi2Byte, fmi2ValueReference, fmi2FMUstate, fmi2Component, fmi2ComponentEnvironment, fmi2Enum
 
+const fmi2Status = Cuint
```

Comment:
why is the declaration seperated from the doc string now? 
(doc string starts in L44, declaration is in L36)

Reply:
because otherwise each status would need its own docstring. Even if that option is preferred, I recommend to implement it in the future and go with this solution for now